sale_unreconciled/views/sale_order_view.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="sale_order_form" model="ir.ui.view">
+        <field name="name">sale.order.form - sale_unreconciled</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <field name="payment_term_id" position="after">
+                <field name="unreconciled" invisible="1" />
+            </field>
+            <xpath expr="//button[@name='action_view_invoice']" position="after">
+                <field name="picking_ids" invisible="1" />
+                <button
+                    type="object"
+                    string="Unreconciled Journal Items"
+                    name="action_view_unreconciled"
+                    groups="account.group_account_manager"
+                    class="oe_stat_button"
+                    icon="fa-gears"
+                    attrs="{'invisible':[('unreconciled', '=', False)]}"
+                >
+                </button>
+                <button
+                    type="object"
+                    string="Reconcile"
+                    groups="account.group_account_manager"
+                    name="action_reconcile"
+                    class="oe_stat_button"
+                    icon="fa-link"
+                    attrs="{'invisible':['|', ('unreconciled', '=', False), ('state', '!=', 'done')]}"
+                >
+                </button>
+            </xpath>
+        </field>
+    </record>
+    <record id="view_sale_order_filter" model="ir.ui.view">
+        <field name="name">sale.quotation.select - sale_unreconciled</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_sales_order_filter" />
+        <field name="arch" type="xml">
+            <filter name="my_sale_orders_filter" position="after">
+                <filter
+                    name="unreconciled"
+                    string="Unreconciled"
+                    domain="[('unreconciled','=', True)]"
+                    groups="account.group_account_manager"
+                    help="sale orders with unreconciled journal items."
+                />
+                <filter
+                    name="finance_review"
+                    string="To Review By Finance"
+                    domain="[('unreconciled','=', True), ('invoice_status', '=', 'invoiced'), ('is_delivered', '=', True)]"
```

Comment:
This sounds like a more specific customization

Reply:
Actually, I think this filter would be the most useful utility in this module, as long as the "Unreconciled " filter will show sales that are unreconciled and they are correct. In any case I am removing this because the field is_delivered is removed as well.

Reply:
Removed the filter, it is a custom thing that is not necessary to be in the module